plugins/Android/install.sh

```diff
@@ -1,51 +1,128 @@
-#!/bin/bash -e
+#!/bin/bash
+set -euo pipefail
 
 # directories
 CWD=`dirname $0`
 CWD=`cd $CWD && pwd -P`
 
-BUILD_DIR="${CWD}/webview"
-LIBS_DIR="${BUILD_DIR}/libs"
+case $(uname) in
+Darwin)
+    export JAVA_HOME='/Applications/Unity/Hub/Editor/2022.3.32f1/PlaybackEngines/AndroidPlayer/OpenJDK'
+    export ANDROID_SDK_ROOT='/Applications/Unity/Hub/Editor/2022.3.32f1/PlaybackEngines/AndroidPlayer/SDK'
+    export PATH=$ANDROID_SDK_ROOT/platform-tools:$ANDROID_SDK_ROOT/tools:$ANDROID_SDK_ROOT/tools/bin:$JAVA_HOME/bin:$PATH
+    ;;
+MINGW64_NT*)
+    export JAVA_HOME='/c/PROGRA~1/Unity/Hub/Editor/2022.3.32f1/Editor/Data/PlaybackEngines/AndroidPlayer/OpenJDK'
+    export ANDROID_SDK_ROOT='/c/PROGRA~1/Unity/Hub/Editor/2022.3.32f1/Editor/Data/PlaybackEngines/AndroidPlayer/SDK'
+    export PATH=$ANDROID_SDK_ROOT/platform-tools:$ANDROID_SDK_ROOT/tools:$ANDROID_SDK_ROOT/tools/bin:$JAVA_HOME/bin:$PATH
+    ;;
+esac
+DEST_DIR='../../build/Packager/Assets/Plugins/Android'
+
+if [ ! -d "$JAVA_HOME" ]
+then
+    echo 'Looking for Java Home'
+    echo 'From Unity Hub, please install 2022.3.32f1 with the android module.'
+    exit 1
+fi
 
 # options
+TARGET="webview"
 MODE="Release"
 SCRIPTING_BACKEND="mono"
-UNITY="/Applications/Unity/Hub/Editor/2021.3.23f1"
+UNITY="2022.3.32f1"
 for OPT in $*
 do
     case $OPT in
-    '--release' )
-        MODE="Release"
-        ;;
-    '--develop' )
-        MODE="Develop"
+    '--nofragment')
+        TARGET="webview-nofragment"
         ;;
-    '--il2cpp' )
-        SCRIPTING_BACKEND="il2cpp"
+    '--development')
+        MODE="Development"
         ;;
-    '--mono' )
-        SCRIPTING_BACKEND="mono"
+    '--zorderpatch')
+        UNITY='2022.3.32f1'
```

Comment:
we dont need the `zorderpatch` flag for us it is always the same unity

Reply:
I didn't touch this, only the unity version.

Reply:
ok, revert to what gree had, one less conflict in the future.
We are not passing this flag anyway